File: AndroidSDK/Basic/Lesson_1/HelloActivity/app/src/main/java/ru/android/helloactivity/MainActivity.java
package ru.android.helloactivity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
//        String s = getResources().getString(R.string.hello_geekbrains);
//        setContentView(R.layout.activity_main);
//
//        TextView greeting = findViewById(R.id.greeting);
//
//        String hello = getResources().getString(R.string.hello_geekbrains);
//        hello += "!!";
//        greeting.setText(hello);

    }
}

File: AndroidSDK/Kotlin/Weatherforstudying/app/src/main/java/ru/dmkalvan/weatherforstudying/ui/MainActivity.kt
package ru.dmkalvan.weatherforstudying.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ru.dmkalvan.weatherforstudying.R
import ru.dmkalvan.weatherforstudying.databinding.ActivityMainBinding
import ru.dmkalvan.weatherforstudying.ui.citylist.CityListFragment

class MainActivity : AppCompatActivity() {

    lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        if (savedInstanceState == null){
            supportFragmentManager.beginTransaction()
                .replace(R.id.container, CityListFragment.newInstance())
                .commitAllowingStateLoss()
        }
    }
}